app.js

```diff
@@ -2,6 +2,21 @@ const express = require('express');
 const bodyParser = require('body-parser');// initialize our express app
 const app = express();
 
+app.use(function(req, res, next) {
+  var allowedOrigins = ['http://localhost:3000', ' https://climatetree.azurewebsites.net'];
```

Comment:
this is a great place to use env vavriables. Instead of hard coding it in code.

Reply:
Yup. I can do that.